src/ua/hillel/StringReverse.java

```diff
@@ -0,0 +1,20 @@
+package ua.hillel;
+
+public class StringReverse {
+    public static void main(String[] args) {
+
+        stringReverse();
+    }
+
+    public static void stringReverse() {
```

Comment:
The method should return a `String` not a `void`.
It should accept a parameter `String source` that will be reversed here in your method and returned as a result.